pkg/minikube/cluster/cluster_windows_test.go

```diff
@@ -36,16 +36,9 @@ func TestCreateHypervHostGeneratesCorrectIsoUrl(t *testing.T) {
 	isoPath := filepath.Join(constants.Minipath, "cache", "iso", "boot2docker.iso")
 	expectedURL := "file://" + filepath.ToSlash(isoPath)
 
-	d := createHypervHost(machineConfig)
-	expectedDriver := "*hyperv.Driver"
+	options, _ := createHypervHost(machineConfig)
 
-	if reflect.TypeOf(d).String() != expectedDriver {
-		t.Fatalf("Unexpected driver type. Expected '%s' but got '%s'", expectedDriver, reflect.TypeOf(d).String())
-	}
-
-	driver := d.(*hyperv.Driver)
-
-	if driver.Boot2DockerURL != expectedURL {
+	if options.String("") != expectedURL {
```

Comment:
Another reason of failure is here. I think you missed `options.String("hyperv-boot2docker-url")`